### PR TITLE
Added new birthday field to employees table

### PR DESCRIPTION
**Pull request by Iulia-Ionita**
@AdrianFlorian - please check if this is all we need for a new field

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -38,6 +38,7 @@
     t.boolean  "is_active"
     t.boolean  "is_working_from_office"
     t.boolean  "is_working_daily"
+    t.datetime "birthday"
```
Comment (AdrianFlorian) [inline db/schema.rb]: to add a new field to a model you'll need to generate a migration `rake g migration add_birthday_to_employee birthday:datetime`, then update the _form.html.haml as well.

Comment (AdrianFlorian) [reply db/schema.rb]: see https://stackoverflow.com/questions/4834809/adding-a-column-to-an-existing-table-in-a-rails-migration

Comment (Iulia-Ionita) [reply db/schema.rb]: Done 👍

**Comment by AdrianFlorian** [comment]
👏 👏 going live now :-)

**Pull request merged**